test/e2e/support/utils.js

```diff
@@ -10,7 +10,7 @@ import { URL } from 'url';
 import { times, castArray } from 'lodash';
 
 const {
-	WP_BASE_URL = 'http://localhost:8889',
+	WP_BASE_URL = 'http://localhost:8888',
```

Comment:
In this huge diff I missed this change, but we shouldn't be doing this. We have a separate test environment for a reason 😄

Reply:
Sorry that was a debug change :), I would have expected the tests to fail because of this change :P

Reply:
Yeah, I'm a bit surprised they didn't, but actually the point is to have a fresh state that's always reset for the tests… it might be that because the tests on Travis are fresh no matter what it worked. I'll see about adding a check for this that at least warns us in my fix?

Reply:
Good catch.